conf/configure.fv3.orion.intel

```diff
@@ -0,0 +1,187 @@
+## Compiler: Intel with IntelMPI
+
+SHELL=/bin/sh
+
+################################################################################
+## Include the common configuration parts
+
+ifdef InNemsMakefile
+include $(TOP)/conf/configure.nems.NUOPC
+endif
+
+############
+# commands #
+############
+FC = mpiifort
+CC = mpiicc
+CXX = mpiicpc
+LD = mpiifort -mkl=sequential
+
+#########
+# flags #
+#########
+# default is 64-bit OpenMP non-hydrostatic build using AVX2
+DEBUG =
+REPRO =
+VERBOSE =
+OPENMP = Y
+AVX2 = Y
+HYDRO = N
+CCPP = N
+STATIC = N
+
+include       $(ESMFMKFILE)
+ESMF_INC    = $(ESMF_F90COMPILEPATHS)
+
+NEMSIOINC = -I$(NEMSIO_INC)
+NCEPLIBS = $(POST_LIB) $(NEMSIO_LIB) $(G2_LIB4) $(G2TMPL_LIB) $(BACIO_LIB4) $(SP_LIBd) $(W3EMC_LIBd) $(W3NCO_LIBd) $(CRTM_LIB) $(PNG_LIB) $(JASPER_LIB) $(Z_LIB)
+
+##############################################
+# Need to use at least GNU Make version 3.81 #
+##############################################
+need := 3.81
+ok := $(filter $(need),$(firstword $(sort $(MAKE_VERSION) $(need))))
+ifneq ($(need),$(ok))
+$(error Need at least make version $(need).  Load module gmake/3.81)
+endif
+
+NETCDF_ROOT = $(NETCDF)
+INCLUDE = -I$(NETCDF_ROOT)/include
+NETCDF_INC = -I$(NETCDF_ROOT)/include
+ifneq ($(findstring netcdf/4,$(LOADEDMODULES)),)
+  NETCDF_LIB += -L$(NETCDF)/lib -lnetcdff -lnetcdf
+else
+  NETCDF_LIB = -L$(NETCDF)/lib -lnetcdff -lnetcdf
+endif
+
+FPPFLAGS := -fpp -Wp,-w $(INCLUDE)
+CFLAGS := $(INCLUDE)
+
+FFLAGS := $(INCLUDE) -fno-alias -auto -safe-cray-ptr -save-temps -ftz -assume byterecl -nowarn -sox -align array64byte
+
+CPPDEFS += -Duse_libMPI -Duse_netCDF -DSPMD -DUSE_LOG_DIAG_FIELD_INFO -DUSE_GFSL63 -DGFS_PHYS -Duse_WRTCOMP
+CPPDEFS += -DNEW_TAUCTMAX -DINTERNAL_FILE_NML
+
+ifeq ($(HYDRO),Y)
+CPPDEFS +=
+else
+CPPDEFS += -DMOIST_CAPPA -DUSE_COND
+endif
+
+ifeq ($(NAM_phys),Y)
+CPPDEFS += -DNAM_phys
+endif
+
+ifeq ($(32BIT),Y)
+CPPDEFS += -DOVERLOAD_R4 -DOVERLOAD_R8
+FFLAGS += -i4 -real-size 32
+else
+ifeq ($(REPRO),Y)
+FFLAGS += -i4 -real-size 64
+else
+FFLAGS += -i4 -real-size 64 -no-prec-div -no-prec-sqrt
+endif
+endif
+
+ifeq ($(REPRO),Y)
+FFLAGS += -qno-opt-dynamic-align
+CFLAGS += -qno-opt-dynamic-align
+else
+ifeq ($(AVX2),Y)
+# Don't use the AVX512 flags yet on hera
+#FFLAGS += -axSSE4.2,AVX,CORE-AVX2,CORE-AVX512 -qno-opt-dynamic-align
+#CFLAGS += -axSSE4.2,AVX,CORE-AVX2,CORE-AVX512 -qno-opt-dynamic-align
+FFLAGS += -xCORE-AVX2 -qno-opt-dynamic-align
+CFLAGS += -xCORE-AVX2 -qno-opt-dynamic-align
+else
+FFLAGS += -qno-opt-dynamic-align
+CFLAGS += -qno-opt-dynamic-align
+endif
+endif
+
+ifeq ($(MULTI_GASES),Y)
+CPPDEFS += -DMULTI_GASES
+endif
+
+FFLAGS_OPT = -O2 -debug minimal -fp-model source -qoverride-limits -qopt-prefetch=3
+FFLAGS_REPRO = -O2 -debug minimal -fp-model consistent -qoverride-limits -g -traceback
+FFLAGS_DEBUG = -g -O0 -check all -check noarg_temp_created -check nopointer -warn -warn noerrors -fp-stack-check -fstack-protector-all -fpe0 -debug -traceback -ftrapuv
```

Comment:
It looks like at commit  a59b6cf we lost the additional debug flag -link_mpi=dbg in the hera.intel configuration; that has now propagated forward to the orion configuration. 

Since adding this back might cause the debug test to fail, I will create an issue for adding this setting back to both the hera.intel and orion.intel.